modules/apps/frontend-theme/frontend-theme-classic/src/css/_custom.scss

```diff
@@ -1,4 +1,3 @@
-// @import 'components/group-item'; // To Verify
```

Comment:
Not relevant change

![list-group-item-flex-2](https://user-images.githubusercontent.com/46778114/107658057-0fb6fc80-6c86-11eb-8079-0394ad00ca00.jpg)
![list-group-item-flex1](https://user-images.githubusercontent.com/46778114/107658068-12195680-6c86-11eb-90a3-1b6878166d9c.jpg)